### DISCUSSION TITLE
How to use 2d images in 3d slicer for segmentation?

### DISCUSSION THREAD

**Discussion by Suchi97** [Project-MONAI/MONAILabel#851] [General]
Hi Team, 

Can you please guide me how to use 2d images in 3d slicer.

I tried to use some png files and annotated using segment editor. But the labels are saved in nifti format. 
So is there a way to save the labels in png format.

Any help is appreciated
Thanks

**Comment by diazandr3s** [comment]
Hi @Suchi97,

Thanks for opening this discussion. 
MONAI Label was developed for 3D image segmentation. However, there are no blockers to make it work for 2D segmentation.
If you used 3D Slicer as the viewer, it might be easier to first convert the images in nifti format. 

We've started exploring CVAT (https://github.com/Project-MONAI/MONAILabel/tree/main/plugins/cvat) as a viewer as well, but this is the very early stages.

We welcome contributions from the community :)

**Reply by Suchi97** [reply]
Hi @diazandr3s ,

Thanks for your reply.
I will try with CVAT tool. 

Thanks again for your help
